canonical-web-and-design/ubuntu.com
# Add search to all ubuntu.com docs

## Done

Search for all ubuntu.com docs

## QA

Theses Checks:
- Search something that yields many results e.g. application and click the next and previous pagination links
- Search something that don't yield results e.g. asdf and check links are correct


In the following URLs:

https://ubuntu-com-9375.demos.haus/ceph/docs
https://ubuntu-com-9375.demos.haus/server/docs
https://ubuntu-com-9375.demos.haus/core/docs
https://ubuntu-com-9375.demos.haus/openstack/docs


## Issue / Card

Fixes #9049 #9050 #9051 #9381
